### ISSUE TITLE
Tab navigation shortcuts work

### ISSUE THREAD

**Issue by gsbhasin123** [peppermintos/ice#58]
Ctrl T/W, Ctrl Tab, etc...

**Comment by jhabarsingh** [comment]
can someone please explain me the issue properly .

**Comment by codic12** [comment]
the issue is that you can use ctrl t to open a new tab, ctrl w to close a tab, ctrl tab to change tab...

**Comment by jhabarsingh** [comment]
sir in my case it is working.

**Comment by codic12** [comment]
Yes, I am saying that these shortcuts should _not_ work

**Comment by jhabarsingh** [comment]
aren't these standard shortcuts .

**Comment by codic12** [comment]
Yes, these shortcuts are standard for a web browser, but ice is supposed to create SSBs which behave like apps. So you should not be able to create tabs, switch tabs, etc.

**Comment by jhabarsingh** [comment]
@codic12, when we install Progressive web apps on our desktop they also provide these shortcuts, so I think these shortcuts should be present

**Comment by codic12** [comment]
progressive web apps, say with Chromium's feature, do NOT provide these

**Comment by jhabarsingh** [comment]
@codic12  I tried to search about this but I couldn't find. can you send me the link where they have mentioned this

**Comment by codic12** [comment]
could not find what? the chromium feature? first thing i found (this is for edge-chromium but also applies to chromium and chrome) https://www.groovypost.com/howto/using-web-apps-new-chromium-edge-windows-10/

**Comment by twizzay-code** [comment]
Please let this feature stay. It is very useful when popups occur.

In my use-case I use Ice for more than just application type sites. I use it to containerize browsers for specific sites that I don't trust to not track me around the internet.

**Comment by grafiksinc** [comment]
@twizzay-code  - This repo is abandoned....and ICE is nolonger developed you can read about why and what is replacingit 
Peppermint as anorganization  has moved all code to "CoderBerg.org"
https://sourceforge.net/p/peppermintos/pepos/general/thread/266fd4bb8f/?limit=25#7e5f/4b53
